### PR TITLE
feat: add unit test constraints to review agent

### PR DESCRIPTION
**Pull request by JackYPCOnline**
*Issue #, if available:*
https://github.com/strands-agents/sdk-typescript/issues/767

*Description of changes:*

Add more constaints to test review.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: strands-command/agent-sops/task-reviewer.sop.md [hunk 1]
```diff
@@ -101,6 +101,8 @@ Analyze the test coverage and quality of tests.
 - You MUST check for proper test organization and naming
 - You MUST identify missing edge cases or error scenarios
 - You MUST verify integration tests are included when appropriate
+- You MUST flag tests that assert on individual fields when the full object or shape can be asserted in a single equality check, since per-field assertions silently miss unexpected or regressed fields
+- You MAY accept per-field assertions only when a field is non-deterministic or irrelevant to the behavior under test, and the test isolates that field rather than splitting the whole assertion
```
Comment (notowen333) [inline strands-command/agent-sops/task-reviewer.sop.md:105]: nit: 2 sentences would be clearer directive

**Review by notowen333 — APPROVED** [review]
